executor/simple.go

```diff
@@ -998,6 +1027,15 @@ func (e *SimpleExec) executeAlterUser(ctx context.Context, s *ast.AlterUserStmt)
 			}
 		}
 
+		if len(lockAccount) != 0 {
+			_, _, err := exec.ExecRestrictedSQL(ctx, nil,
+				`UPDATE %n.%n SET account_locked=%? WHERE Host=%? and User=%?;`,
+				mysql.SystemDB, mysql.UserTable, lockAccount, strings.ToLower(spec.User.Hostname), spec.User.Username)
```

Comment:
I think `spec.User.Hostname` should already be lowered in the parser? If not, no problem to do it here of course.

Reply:
Yes, the `ToLower` is unnecessary :-) FIxed it.